nservicebus/upgrades/support-policy.md

```diff
@@ -33,3 +50,28 @@ NServiceBus versions are wire-compatible, meaning the endpoints using different
 However, some features might require data migration (e.g. converting from old to new format), and the conversion might be performed in the background after upgrade.
 
 Therefore the recommended approach is to upgrade 1 major version at a time, including a full regression testing of the system and deployment to production.
+
+
+## FAQ
+
+**Does this policy apply to the [ServiceControl](/servicecontrol/), [ServiceInsight](/serviceinsight/), or [ServicePulse](/servicepulse/) applications?**
+
+No, the policy applies only to the supported NServiceBus packages listed on the [NServiceBus Packages Supported Versions](supported-versions.md) page.
+
+**Which version of NServiceBus should be used to start a new project?**
+
+It's preferable to use the newest major version of NServiceBus when starting a new project. The support policy guarantees that the major version will be supported for at least 2 years after it is replaced by a new major version.
+
+**Which version of NServiceBus should be used when adding to an existing project?**
+
+When adding new functionality to an existing project, it's generally easiest to use the same version of NServiceBus already in use by other endpoints, as long as that version of NServiceBus is still supported. It is also possible to use a newer version of NServiceBus in a specific endpoint as a step towards [upgrading](#upgrading) the entire system.
```

Comment:
I think "all other endpoints is misleading". If the code for endpoints do not share a common lib that refs nsb, and do not exist in the same sln, then it is fine for them use diff versions of nsb